src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -133,18 +136,45 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
 
     with self.argument_context('ams content-key-policy') as c:
         c.argument('account_name', account_name_arg_type)
-        c.argument('content_key_policy_name', name_arg_type)
+        c.argument('content_key_policy_name', name_arg_type,
+                   help='The content key policy name.')
         c.argument('description', help='The content key policy description.')
         c.argument('clear_key_configuration',
                    action='store_true',
-                   arg_group='Basic Policy Options',
+                   arg_group='Clear Key Configuration (AES Encryption)',
                    help='Use Clear Key configuration, a.k.a AES encryption. It\'s intended for non-DRM keys.')
         c.argument('open_restriction',
                    action='store_true',
-                   arg_group='Basic Policy Options',
-                   help='Use open restriction. License or key will be delivered on every request.')
-        c.argument('policy_option_name',
-                   help='The name of the policy option.')
+                   arg_group='Open Restriction',
+                   help='Use open restriction. License or key will be delivered on every request. Not recommended for production environments.')
+        c.argument('policy_option_name', help='The content key policy option name.')
+        c.argument('policy_option_id', help='The content key policy option identifier.')
+        c.argument('issuer', arg_group='Token Restriction', help='The token issuer.')
+        c.argument('audience', arg_group='Token Restriction', help='The audience for the token.')
+        c.argument('token_key', arg_group='Token Restriction', help='Either a string (for symmetric key) or a filepath to a certificate (x509) or public key (rsa). Must be used in conjunction with --token-key-type.')
+        c.argument('token_key_type', arg_group='Token Restriction', help='The type of the token key to be used for the primary verification key. Allowed values: {}'.format(", ".join(get_token_completion_list())))
+        c.argument('alt_token_key', arg_group='Token Restriction', help='Either a string (for symmetric key) or a filepath to a certificate (x509) or public key (rsa). Must be used in conjunction with --alt-token-key-type.')
```

Comment:
Add --add-alt-token-key & --add-alt-token-key-type arguments for `option update` command. Add a friendly description to let the user know he is adding a new alternate key